torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -2205,7 +2205,9 @@ at::Tensor XLANativeFunctions::nan_to_num(const at::Tensor& self,
   if (!at::native::is_floating_point(self)) {
     return CopyTensor(self);
   }
-  auto element_type = TensorTypeToRawXlaType(self.scalar_type());
+  XLATensor input_tensor = bridge::GetXlaTensor(self);
```

Comment:
nit, replace the `bridge::GetXlaTensor(self)` down below to be `input_tensor`.

Reply:
done.